mod_bot/main.py

```diff
@@ -56,6 +56,18 @@ def add_minutes(hstart, mstart, mn):
     print(conf)
     is_connected = True
 
+
+def clean_spaces(command):
+    return clean_char(command, ' ')
+
+
+def clean_at_sign(command):
+    return clean_char(command, '@')
+
+
+def clean_char(command, char):
+    return list(filter(lambda part: part.trim() != char, command))
```

Comment:
strip() ? trim method does not exist in python3

Reply:
will write some tests I guess